docs/get_started/examples/index.md

```diff
@@ -0,0 +1,321 @@
+# Overview
+
+These example workflows constitutes an introduction to the HPE Nimble Storage SDK for Python. It may be used to get familiar both HPE Nimble Storage and NimbleOS as a product as well as getting familiar with the SDK.
+
+The examples are available to [browse on GitHub](https://github.com/hpe-storage/nimble-python-sdk/tree/master/workflows).
+
+[TOC]
+
+## Setup
+
+Ensure Python 3.6+, `pip` and `git` is installed on your workstation. 
+
+```markdown
+pip install nimble-sdk
+git clone https://github.com/hpe-storage/nimble-python-sdk
+cd nimble-python-sdk/workflows
+```
+
+It's possible to run the example scripts interactively, but it's more convenient to configure the accompanied `workflow_config.json`:
+
+```json
+{
+  "hostname":"192.168.1.100",  # Set these to point to the array being used
+  "username":"admin",
```

Comment:
shall we mark it as ******

Reply:
Googling "nimble default password" comes up as a featured snippet for me, so I wouldn't be too worried. The IP address is made up from a RFC1918 class C network.